src/main/scala/com/microsoft/hyperspace/actions/CreateAction.scala

```diff
@@ -35,7 +35,8 @@ class CreateAction(
     dataManager: IndexDataManager)
     extends CreateActionBase(dataManager)
     with Action {
-  final override def logEntry: LogEntry = getIndexLogEntry(spark, df, indexConfig, indexDataPath)
+  final override def logEntry: LogEntry =
+    getIndexLogEntry(spark, df, indexConfig, indexDataPath, super[Action].endId)
```

Comment:
nit:
```suggestion
    getIndexLogEntry(spark, df, indexConfig, indexDataPath, super.endId)
```
passing Action seems unnecessary